test/backup.py

```diff
@@ -246,11 +257,34 @@ def _remove_backup(self, backup):
         ['RemoveBackup', 'test_keyspace/0', backup],
         auto_log=True, mode=utils.VTCTL_VTCTL)
 
+  def _backup_only(self, t, tablet_type='replica'):
+    """Erase mysql/tablet dir, then start tablet with restore only."""
+    logging.debug('starting backup only tablet %s',t.tablet_alias)
+    self._reset_tablet_dir(t)
+
+    logging.debug('building command line')
+
+    xtra_args = ['-db-credentials-file', db_credentials_file,'-backup_and_exit']
+    if use_xtrabackup:
+      xtra_args.extend(xtrabackup_args)
+
+    proc = t.start_vttablet(wait_for_state=False,
+                     init_tablet_type=tablet_type,
+                     init_keyspace='test_keyspace',
+                     init_shard='0',
+                     supports_backups=True,
+                     extra_args=xtra_args)
+    logging.debug('tablet started waiting for process to end %s',proc)
+    utils.wait_procs([proc],True)
+
   def test_backup_rdonly(self):
-    self._test_backup('rdonly')
+    self._test_backup('rdonly', False)
 
   def test_backup_replica(self):
-    self._test_backup('replica')
+    self._test_backup('replica', False)
+
+  def test_tablet_backup_only(self):
+    self._test_backup('replica', True)
```

Comment:
We would ideally not want this "fake" tablet to serve queries, because then it's potentially disruptive when we take it away unexpectedly. Maybe we could make some small changes to allow `init_tablet_type=backup` so it never tries to become anything else?